setup.py

```diff
@@ -47,6 +47,6 @@
     keywords='MRI high-resolution laminar subcortex',
     packages=find_packages(),
     include_package_data=True,
-    install_requires=['numpy', 'nibabel', 'psutil','antspyx'],
+    install_requires=['psutil', 'antspyx'],
```

Comment:
`numpy` and `nibabel` are dependencies already of `antspyx`

Reply:
yes, but.. if we were to drop antspyx, we would still need them. Better to keep them explicitly?

Reply:
That's fair. Will bring them back and pin their version number.